source/plugin/permissions.rst

```diff
@@ -2,9 +2,225 @@
+
+The following example could be used to check whether the Player is allowed to execute the teleport command.
+
+.. code-block:: java
+    
+    public canTeleport(Player subject, World targetWorld) {
+        return subject.hasPermission("myPlugin.command.teleport.execute") 
+                && (subject.getWorld() == targetWorld
+                        || subject.hasPermission("myPlugin.command.teleport." + targetWorld.getName()));
+    }
+
+Inheritance
+~~~~~~~~~~~
+
+If a ``Subject`` got a permission assigned, it will use that value.
+Otherwise it will be inherited from any parent ``Subject``. It does not matter what kind of parent (ex group or player)
+``Subject`` that might be.
+
+If neither the subject itself nor any parent subjects grant or deny a permission then it will be inherited from the
+default ``Subject``. The default subject can be obtained from the ``PermisionService``.
+It is recommended for plugins to define their own permissions to the default's transient ``SubjectData`` during every
+server start-up. This allows the server owner to overwrite the defaults defined by plugins according to his needs using
+the default's persistent ``SubjectData``.
+
+.. note::
+
+    Its only specified for the default ``Subject`` that the persistent ``SubjectData`` takes precedence over transient
+    ones.
+
+If neither the Subject, nor any of its parents, nor the default grant or deny a permission, then it is automatically
+denied.
 
 .. note::
 
-    This page has not been filled yet by our Documentation Team. If you feel like you can help, you can do so on `our
-    GitHub repository <https://github.com/spongepowered/spongedocs>`_. Also see the `related GitHub Issue
-    <https://github.com/SpongePowered/SpongeDocs/issues/32>`_ for more information on what is required.
+    Order of precedence in descending order:
+    
+    * Subject itself
+    * Parent Subjects
+    * Defaults
+        * Persistent
+        * Transient
+    * Deny permission
+
+SubjectCollections
+==================
+
+A container for subjects that can be used obtained a Subject by name.
+These are the default Subject Collections:
+
+* User
+    * Contains all on-line and off-line ``User`` / ``Player`` that exist.
+* Group 
+    * Contains all group ``Subject``. Groups are a simple way of structuring a ``Subject's`` inheritance tree.
+* System
+    * Contains ``Subjects`` required by the system. TODO: What is their special purpose/use case. 
+* Command Block
+    * Contains all ``Subjects`` for command blocks. These are useful if you would like to run a ``CommandBlock`` only
+      with the permissions of the creator.
+* Role Template
+    * Contains all role template subjects that are used in ``PermissionDescriptions``. Useful to lookup all recommended
+      permissions for a user. These should not be used for inheritance.
+
+SubjectData
+===========
+
+SubjectData are the actual permission store connected to the Subject. There are two types of Subject stores. The
```

Comment:
I'd use a colon, ie. "Subject stores:".
Additionally, a quibble: should we be saying "saved to disk" or "saved to file"? Maybe it's easier just to say "saved" and stay system agnostic.